tests/stress/taskStress_test.go

```diff
@@ -34,39 +36,45 @@ func TestTaskStress(t *testing.T) {
 		"TaskJ": 1,
 	}
 
-	for taskName, copies := range taskNameCopies {
-		go func(taskName string, copies int) {
-			log.Debugf("Creating task %s (copies: %d)", taskName, copies)
-			s.Layer0.CreateTask(taskName, environmentID, deployID, copies, nil)
-		}(taskName, copies)
+	for taskName := range taskNameCopies {
```

Comment:
`taskNameCopies` doesn't make sense anymore, this should be re-created to have the same behavior as it did before (100 tasks I think?)

Reply:
Yes, but that will be part of a future PR that will be responsible for fixing the system tests.

Reply:
👍 sounds good